nuget/helpers/lib/NuGetUpdater/Directory.Common.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <CommonTargetFramework>net8.0</CommonTargetFramework>
+    <ArtifactsDir>$(MSBuildThisFileDirectory)artifacts</ArtifactsDir>
+    <BaseOutputPath>$(ArtifactsDir)\bin\$(MSBuildProjectName)</BaseOutputPath>
+    <BaseIntermediateOutputPath>$(ArtifactsDir)\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
```

Comment:
You can get this sort of behaviour with just the new `UseArtifactsOutput` property added in the .NET 8 SDK: https://learn.microsoft.com/dotnet/core/sdk/artifacts-output

Reply:
I didn't know this existed, thank you!